testing/transcripts/package.json

```diff
@@ -9,14 +9,14 @@
     "author": "Microsoft Corp.",
     "license": "MIT",
     "dependencies": {
-        "@types/node": "^10.17.27",
-        "@types/restify": "^8.4.2",
+        "@types/node": "^22.5.0",
```

Comment:
We discussed this. We are going to use @types/node 18.

Reply:
fixed